flink-runtime/src/main/java/org/apache/flink/runtime/dispatcher/DispatcherJob.java

```diff
@@ -165,17 +171,10 @@ private DispatcherJob(
 			if (isInitialized()) {
 				return getJobMasterGateway().thenCompose(jobMasterGateway -> jobMasterGateway.cancel(timeout));
 			} else {
-				log.info("Cancellation during initialization requested for job {}. Job will be cancelled once JobManager has been initialized.", jobId);
+				log.info("Cancellation during initialization for job {}.", jobId);
```

Comment:
Since it is quite rare to cancel jobs during initialization, I don't believe this log message will hurt

Reply:
That's not really an answer to my question, and imo further reinforces that it shouldn't be logged on info.

Reply:
Let me try again: I believe it is relevant enough because we are interrupting the initialization thread, potentially leading to all kinds of weird errors. I should change the log message a bit to make that more explicit.